File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/PokemonMoveAccuracyBoosterModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import com.sfh.pokeRogueBot.model.modifier.ChooseModifierItem;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class PokemonMoveAccuracyBoosterModifierItem extends PokemonHeldItemModifierItem implements ChooseModifierItem {

    public static final String TARGET = "PokemonMoveAccuracyBoosterModifierType";

    private int amount;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/PokemonMoveAccuracyBoosterModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

class PokemonMoveAccuracyBoosterModifierItem : PokemonHeldItemModifierItem() {

    companion object {
        const val TARGET = "PokemonMoveAccuracyBoosterModifierType"
    }

    var amount: Int = 0
}